pages/index.vue

```diff
@@ -8,6 +8,12 @@
 </template>
 
 <script setup lang="ts">
+import {
+  useSchemaOrg,
+  defineWebSite,
+  defineWebPage,
+  defineOrganization,
+} from "@unhead/schema-org-vue";
```

Comment:
Could you double check if you need these imports?

Reply:
ooops, I added it before doing npm ci and it's not needed anymore, thanks for noticing!